common/app/views/fragments/atoms/youtube.scala.html

```diff
@@ -10,10 +10,27 @@
     ))">
 
         @defining(if(!host.isEmpty) "&origin=" + host else "") { origin =>
-            <iframe id="youtube-@media.assets.head.id" width="100%" height="100%"
+            <iframe class="atom--media--youtube--iframe" id="youtube-@media.assets.head.id" width="100%" height="100%"
             src="https://www.youtube.com/embed/@media.assets.head.id?modestbranding=1&enablejsapi=1&rel=0&showinfo=0@origin" frameborder="0"
             allowfullscreen="">
             </iframe>
+
+            @media.posterUrl.map { image =>
+                <div class="atom--media--youtube--overlay">
```

Comment:
I also wanted to check about the classnames. Based on these it seems that `media` is a modifier on `atom` and `overlay` is a modifier of `atom--media--youtube`. Regarding the former, is there an atom class already that you are modifying? It doesn't appear so in the code. 

I would think the styling for the youtube player would be unique and should have a unique classname as well. Maybe something like `youtube-atom` and overlay, as it is a child of the youtube atom could be `youtube-atom__overlay`.

Let me know what you think. Naming is hard, but essential to keep code readable.